includes/container.php

```diff
@@ -61,7 +61,8 @@ public function getContainer() {
     $locations = array_merge($this->getComposerDirs(), $this->getInstalledDirectories());
     $loader = new ServicesFileLoader($container, new FileLocator($locations));
     $loader->load('commoncli.services.yml');
-    $container->setParameter('autoloader', $this->autoloader);
+    $container->setParameter('autoloader', NULL);
```

Comment:
As we discussed this can be removed and the services.yaml needs to be updated as it will be used as a service from now on.